addons/stock/models/stock_move.py

```diff
@@ -1,4 +1,4 @@
-# -*- coding: utf-8 -*-
+#test_report_forecast_1_mo_counttest_report_forecast_1_mo_counttest_report_forecast_1_mo_counttest_report_forecast_1_mo_count -*- coding: utf-8 -*-
```

Comment:
Glad I'm not the only one miss-clicking the copy/paste of the name when testing :smile: